helm/flink-kubernetes-operator/values.yaml

```diff
@@ -78,3 +78,8 @@ metrics:
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+
+# Set the jvm start up options for webhook and operator
+jvmArgs:
+  webhook: ""
+  operator: ""
```

Comment:
This looks pretty awkward. We should start structuring the helm options a bit more. 
I will probably open a ticket for this.

Reply:
I also feel the helm option is a little mess now 😄, do you have some suggestion for this ? Or let it be improved in your ticket?

Reply:
I will try to work on this in another ticket, we need to clean up many things . It’s ok as it is